lib/assessment.js

```diff
@@ -4,27 +4,61 @@
 // be called with the current element in the array,
 // its index, and the array itself
 // e.g. callback(currentValue, index, array)
+var reduce = function reduce(array, callback, initialValue) {
+  var previousValue;
+  if (initialValue !== undefined) {
+    previousValue = initialValue;
+  } else {
+    array = array.slice();
+    previousValue = array.shift();
+  }
+
+  array.forEach(function(currentValue, index, array) {
+    previousValue = callback(previousValue, currentValue, index, array);
+  });
+
+  return previousValue;
+};
```

Comment:
This is the function that takes the callback. The question wanted you to write the callback that can be passed to `reduce` to make it behave as `Math.min` on an array.